openssl-sys/build.rs

```diff
@@ -164,6 +168,13 @@ fn try_pkg_config() {
     // Otherwise if we're going to windows we probably can't use pkg-config.
     if target.contains("windows-gnu") && host.contains("windows") {
         env::set_var("PKG_CONFIG_ALLOW_CROSS", "1");
+    } else if target.contains("windows-gnu") && host.contains("linux") {
```

Comment:
@alexcrichton @sfackler But on the other hand, if the user gives an `OPENSSL_LIB_DIR`, it will skip the `find_openssl_dir()`; if the user doesn't, why not give it a bold try.

I think what I need is to check the existence of files before setting the `PKG_CONFIG_PATH` just as how `find_openssl_dir()` deals with OpenSSL installed from Homebrew, e.g. check `/usr/x86_64-w64-mingw32/lib/pkgconfig/libssl.pc`.
What do you think?